conf/evolutions.default/1.sql

```diff
@@ -0,0 +1,19 @@
+# --- !Ups
+
+CREATE TABLE IF NOT EXISTS Users(
+id INTEGER PRIMARY KEY AUTOINCREMENT,
+firstName varchar(255) NOT NULL,
+lastName varchar(255) NOT NULL,
+email varchar(255) NOT NULL UNIQUE,
+password varchar(255) NOT NULL,
+userRole varchar(255) NOT NULL,
+authenticated boolean
+);
+
+INSERT INTO Users(LastName, FirstName, Password, Email, userRole, authenticated) VALUES ('Saad', 'Ali', 'spadsdditcom', 'asdasdasd', 'USER' ,1);
```

Comment:
Passwords don't go into the table directly, hash them and use them that way.

Reply:
The `password` string will be hashed string, if that's what you mean

Reply:
```LastName, FirstName, ... ```
Case of these arguments don't match the real table column.